themes/bootstrap3/js/common.js

```diff
@@ -294,6 +294,17 @@ function htmlEncode(value) {
   }
 }
 
+/**
+ * Keyboard and focus controllers
+ * Adapted from Micromodal
+ * - https://github.com/ghosh/Micromodal/blob/master/lib/src/index.js
+ */
+const FOCUSABLE_ELEMENTS = ['a[href]', 'area[href]', 'input:not([disabled]):not([type="hidden"]):not([aria-hidden])', 'select:not([disabled]):not([aria-hidden])', 'textarea:not([disabled]):not([aria-hidden])', 'button:not([disabled]):not([aria-hidden])', 'iframe', 'object', 'embed', '[contenteditable]', '[tabindex]:not([tabindex^="-"])'];
+function getFocusableNodes(container) {
```

Comment:
Would it be better to plug this into the VuFind namespace container rather than creating a new global function? I'm okay with this as-is if there's no more logical place to put it, but I thought we were trying to avoid globals as much as possible.... maybe this is an opportunity to start a VuFind.util module or something.

Reply:
A VuFind.util would be good, but when I looked at the VuFind object yesterday, it seemed to be solely system things and not utility. So I put this here. This won't be a problem in the future of JS modules.

Reply:
My comment wasn't clear: I don't think this function fits with the methods in the VuFind object, so I don't agree with moving it there. I do look forward to a future where we can get all of this out of the global scope!